metadata.rb

```diff
@@ -5,7 +5,7 @@
 license           'Apache-2.0'
 description       'Installs and configures postgresql for clients or servers'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '7.1.0'
+version           '7.1.1'
```

Comment:
Shouldn't have to bump the version here, please revert this.

Reply:
Does that happen automatically with a release?  If not, I'm confused to why it wouldn't need a version bump.

Reply:
This is something we control as maintainers and Berkshelf is smart enough to source the current directory regardless of version for testing. The old even-odd release cycle in Chef hasn't been needed for a while.